teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>(NN), that were described in Talktorial T007 &gt; either without comma or which instead of that (I think)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Indeed!</p>